### ISSUE TITLE
macOS - dyld: Library not loaded: /usr/local/opt/openssl/lib/libssl.1.0.0.dylib [fixed]

### ISSUE THREAD

**Issue by kensoh** [tebelorg/RPA-Python#222]
Hi RPA for Python macOS users!

If you run into above error message during r.init(), please try running below solution and it should fix it.

Issue with some macOS level changes regarding SSL, OpenSSL version changes, Homebrew and stuffs.
The solution is downloading an updated copy of PhantomJS which seems to work well with these changes.

Here's why I've been missing in action and hope to be back soon to look into issues - https://github.com/tebelorg/RPA-Python/issues/144#issuecomment-753566228
And a ***BIG SHOUT OUT*** to Dominik / @fi-do for being a Community Hero here helping other users!! <3

**Comment by kensoh** [comment]
```py
import rpa as r
import os
os.chdir(os.path.expanduser('~') + '/.tagui/src')
r.download('https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-macosx.zip', 'phantomjs.zip')
r.unzip('phantomjs.zip')
os.rename('phantomjs', 'phantomjs_old')
os.rename('phantomjs-2.1.1-macosx', 'phantomjs')
os.system('chmod -R 755 phantomjs > /dev/null 2>&1')
r.init()
```

**Comment by kensoh** [comment]
Implemented in v1.30, available with `pip install rpa --upgrade`
